stagemonitor-core/src/main/java/org/stagemonitor/core/elasticsearch/ElasticsearchClient.java

```diff
@@ -52,6 +53,8 @@
 	private final HttpClient httpClient;
 	private final CorePlugin corePlugin;
 	private final AtomicBoolean elasticsearchAvailable = new AtomicBoolean(false);
+	private final AtomicBoolean elasticsearchHealthy = new AtomicBoolean(false);
+	private String esMajorVersion;
```

Comment:
`Integer esMajorVersion`, so we don't have to parse it on every invocation of `isElasticsearch6Compatible`